cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -106,8 +106,11 @@ func buildDaisyVars(destinationURI string, sourceImage string, sourceDiskSnapsho
 	}
 
 	if sourceDiskSnapshot != "" {
-		varMap["source_disk_snapshot"] = param.GetGlobalResourcePath(
-			"snapshots", sourceDiskSnapshot)
+		varMap["source_disk_snapshot"] = sourceDiskSnapshot //param.GetGlobalResourcePath(			"snapshots", sourceDiskSnapshot)
```

Comment:
Did you intend to leave the commented code?

Reply:
Nope, good catch. Snapshot export needs some work not related to these changes, but is not urgent since we don't expose this functionality as far as I'm aware.